benchopt/runner.py

```diff
@@ -48,6 +49,12 @@ def colorify(message, color=BLUE):
 @mem.cache
 def run_repetition(objective, solver_class, solver_parameters, meta, sample):
 
+    # check if the module caught a failed import
+    module = importlib.import_module(solver_class.__module__)
+    if module.solver_import.failed_import:
```

Comment:
Not sure if we should require to have the `solver_import` or if we should do the suggested change:
```suggestion
    if (hasattr('solver_import', module)
            and module.solver_import.failed_import):
```
Maybe it is better the way you propose as it is more systematic (all solvers have this `safe_import`) but it means that user need to see this even if the have no package to install.

Reply:
Probably better to not require `solver_import`, indeed.